src/general/index.ts

```diff
@@ -228,3 +228,40 @@ export function downloadFile(downloadConfig: { file: File | null; type: string;
     console.error('File download failed - ', err);
   }
 }
+
+
+/**
+ * This method returns the search id from the url. By default it returns the last searchId from the URL.
+ * If you want you can get any searchId from URL by passing index from last value param
+ *
+ * @param {string} url - The url that is entered
+ * @param {number} indexFromLast - The searchId index from last slash in the URL. By default it is the last index.
+ *
+ * @example
+ * ```
+ * getSearchIdFromUrl('https://groww.in/mutual-funds/user/explore')       //explore
+ * getSearchIdFromUrl('https://groww.in/mutual-funds/user/explore', 2)   //mutual-funds
+ * ```
+ *
+ */
+export function getSearchIdFromUrl(url: string, indexFromLast = 0) {
```

Comment:
Name should be getPathVariableFromUrlIndex

Also can we use url package for this?

Reply:
will update the function name. No url package is similar to query string package and in this only we can get query string from the url which will not solve this use case of getting the any path variable index from url

Reply:
Do you have a check for wrong index .... if I give 4 for the url groww.in/mutual-funds, then what will happen?